react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -373,6 +373,9 @@ class Filmstrip extends PureComponent <Props> {
         }
 
         if (_currentLayout === LAYOUTS.TILE_VIEW) {
+            // Fixes case in which context menu overflows and creates a scroll on the whole filmstrip videos pane.
+            const style = { overflow: 'visible' };
```

Comment:
Maybe define the constant outside of the class in order to save some rerenders. WDYT?